src/sdk/clients/decorators/erc7579/index.ts

```diff
@@ -51,9 +51,11 @@ import {
 
 export type Erc7579Actions<TSmartAccount extends SmartAccount | undefined> = {
   accountId: (args?: GetSmartAccountParameter<TSmartAccount>) => Promise<string>
-  installModule: (args: InstallModuleParameters<TSmartAccount>) => Promise<Hash>
+  installModule: (args: InstallModuleParameters<TSmartAccount> & { signatureOverride?: Hex }) => Promise<Hash>
   installModules: (
-    args: InstallModulesParameters<TSmartAccount>
+    args: InstallModulesParameters<TSmartAccount> & {
+      signatureOverride?: Hex
```

Comment:
Can you add signatureOverride to the InstallModulesParameters type instead?

Reply:
Ok I thought is not really related to the module installation, it's related to the user operation so I left it out of that type but sure I can add it if you think is better @joepegler